WPMedia/fusion-news-theme-blocks
# THEMES-891: Non-manual images focal point support

## Description

This PR updates multiple blocks with the new `getFocalFromANS` helper function so that they can properly send focal point info to the `Image` component. The original ticket mentions the gallery block, but gallery images are displayed at their full aspect ratio with no cropping, so the focal point will not matter.

## Jira Ticket

- [THEMES-891](https://arcpublishing.atlassian.net/browse/THEMES-891)

## Acceptance Criteria

- Focal point display is supported on the following blocks:
  - card-list-block
  - extra-large-promo-block
  - large-promo-block
  - lead-art-block
  - medium-promo-block
  - numbered-list-block
  - product-assortment-carousel-block
  - product-featured-image-block
  - results-list-block
  - search-results-list-block
  - simple-list-block
  - small-promo-block
  - top-table-list-block

## Test Steps

1. Checkout this branch `git checkout THEMES-891`
2. If linking the Themes components repo, make sure the latest `arc-themes-release-version-2.2.0` branch is checked out.
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/card-list-block,@wpmedia/extra-large-promo-block,@wpmedia/large-promo-block,@wpmedia/lead-art-block,@wpmedia/medium-promo-block,@wpmedia/numbered-list-block,@wpmedia/product-assortment-carousel-block,@wpmedia/product-featured-image-block,@wpmedia/results-list-block,@wpmedia/search-results-list-block,@wpmedia/simple-list-block,@wpmedia/small-promo-block,@wpmedia/top-table-list-block`
4. Create or load a page with each affected block on it. There are two stories with promo images cloned from each other, but with different focal points set that can be used for testing:
  - `/global/2024/02/23/story-with-focal-point-promo-pyramid/`
  - `/travel/2024/02/16/story-with-focal-point-images/`
5. These can be used for the individual promo blocks.
6. For list blocks, adjust the content queries so they appear. At the time of writing this PR they are the two newest stories on sandbox themesinternal.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-891]: https://arcpublishing.atlassian.net/browse/THEMES-891?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
